### PR TITLE
Ignition Edifice: roadmap, installation, info

### PR DESCRIPTION
**Pull request by chapulina**
Add Edifice to the docs page, with installation instructions from nightlies and source.

Add Dome to the release features, and update other collections.

Also update roadmap for all collections.

Requires:

- [x] For debian nightlies: https://github.com/ignition-tooling/release-tools/pull/309
- [x] For mac binaries: https://github.com/osrf/homebrew-simulation/pull/1173

### CODE REVIEW

File: edifice/install.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Edifice Installation
+
+Ignition Edifice is under active development and will be officially released on
+March 2021. If you'd like to use a stable release, see
+[Citadel](/docs/citadel).
```
Comment (j-rivero) [inline edifice/install.md:5]: Not sure if we want to suggest latest LTS or latest release.

```suggestion
[Dome](/docs/dome).
```

File: edifice/install.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Until the official release, Edifice can be compiled from source or installed
+from nightly debian packages on Linux.
+
+## Binary installation instructions
+
+Nightly builds are available for Linux and macOS.
```
Comment (j-rivero) [inline edifice/install.md:12]:
```suggestion
Nightly builds are available for Linux (Ubuntu Focal and Bionic on amd64) and macOS.
```

File: edifice/install_ubuntu.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Binary Installation on Ubuntu
+
+Dome binaries are provided for Ubuntu Bionic and Focal. All of the Dome
```
Comment (j-rivero) [inline edifice/install_ubuntu.md:3]:
```suggestion
Edifice nightlies are provided for Ubuntu Bionic and Focal. All of the Edifice
```

File: edifice/install_ubuntu.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+binaries are hosted in the osrfoundation repository. To install all of them,
+the metapackage `ignition-dome` can be installed:
```
Comment (j-rivero) [inline edifice/install_ubuntu.md:5]:
```suggestion
the metapackage `ignition-edifice` can be installed:
```

File: edifice/install_ubuntu.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+```bash
+sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
```
Comment (j-rivero) [inline edifice/install_ubuntu.md:8]:
```suggestion
sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-nightly `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-nightly.list'
```

File: edifice/install_ubuntu.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+Head back to the [Getting started](/docs/all/get_started)
+page to start using Ignition!
+
+## Uninstalling binary install
+
+If you need to uninstall Ignition or switch to a source-based install once you
+have already installed the library from binaries, run the following command:
+
+```bash
+sudo apt remove ignition-dome && sudo apt autoremove
```
Comment (j-rivero) [inline edifice/install_ubuntu.md:25]:
```suggestion
sudo apt remove ignition-edifice && sudo apt autoremove
```

**Review by j-rivero — COMMENTED** [review]
An small patch to define support platforms, arm64 are close to be ready, see https://github.com/ignition-tooling/release-tools/issues/297
```diff
diff --git a/dome/install.md b/dome/install.md
index 62fd08f..9d59f69 100644
--- a/dome/install.md
+++ b/dome/install.md
@@ -2,7 +2,7 @@
 
 Dome supports the following platforms:
 
- * Ubuntu Bionic and Focal on amd64/i386
+ * Ubuntu Bionic amd64/arm64/i386 and Focal on amd64/arm64
  * MacOS Mojave and Catalina
      * Ignition currently only works in headless mode
       (GUI does not render; instead of using `ign gazebo fuel.sdf` command, use
```

I found some old references to Dome in Edifice doc but nothing major.

**Comment by chapulina** [comment]
Thanks for catching all that, @j-rivero ! I think I've addressed everything.

File: edifice/install_ubuntu.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+Edifice nightlies are provided for Ubuntu Bionic and Focal. All of the Edifice
+binaries are hosted in the osrfoundation repository. To install all of them,
+the metapackage `ignition-edifice` can be installed:
+
+```bash
+sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
+sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-nightly `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-nightly.list'
+wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
+sudo apt-get update
+sudo apt-get install ignition-dome
```
Comment (j-rivero) [inline edifice/install_ubuntu.md:12]:
```suggestion
sudo apt-get install ignition-edifice
```

**Review by j-rivero — APPROVED** [review]
Found a minor thing but looks really good.

**Pull request merged**